pkg/apis/core/v1beta1/register.go

```diff
@@ -69,6 +69,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&SeedList{},
 		&Shoot{},
 		&ShootList{},
+		&ShootExtensionStatus{},
+		&ShootExtensionStatusList{},
```

Comment:
What about waiting with `v1beta1` promotion for a while and only add it to `v1alpha1` in the beginning (like done for the `ShootState`)?

Reply:
I thought the goal is to phase out the v1beta1 and not add any more resources to it? Did I misunderstand that?

Reply:
Anyways, I am fine with changing it to `v1alpha1` if that is the process